internal/services/redis/cluster.go

```diff
@@ -223,9 +222,11 @@ func ResourceCluster() *schema.Resource {
 func customizeDiffMigrateClusterSize() schema.CustomizeDiffFunc {
 	return func(_ context.Context, diff *schema.ResourceDiff, _ interface{}) error {
 		oldSize, newSize := diff.GetChange("cluster_size")
-		if newSize == 2 {
-			return errors.New("cluster_size can be either 1 (standalone) ou >3 (cluster mode), not 2")
+
+		if newSize == 2 || newSize.(int) > 3 || newSize.(int) == 1 {
```

Comment:
Can this conditional be simplified? I'm not sure to follow what are the cases that are not ok

Reply:
Ah ah yes, this condition is absolutely useless!